### DISCUSSION TITLE
compattamento e modalita scarica trinceee

### DISCUSSION THREAD

**Discussion by marcolino04** [Courseplay/Courseplay_FS22#2209] [Q&A] [answered]
ciao ,come da titolo metterete anche l opzione modalita compattamento? e scaricamento trincee? per il coursepaly saluti

**Comment by Tensuko** [comment, marked as answer]
Compacting and pushgin is already part of CP.
Empty the Bunker is planed for the next year.

**Comment by marcolino04** [comment]
GrazieInviato dal mio Galaxy
-------- Messaggio originale --------Da: Tensuko ***@***.***> Data: 28/12/22  20:58  (GMT+01:00) A: Courseplay/Courseplay_FS22 ***@***.***> Cc: marcolino04 ***@***.***>, Author ***@***.***> Oggetto: Re: [Courseplay/Courseplay_FS22] compattamento e modalita scarica trinceee (Discussion #2209) 
Compacting and pushgin is already part of CP.
Empty the Bunker is planed for the next year.
